src/main/java/org/repodriller/RepositoryMining.java

```diff
@@ -76,6 +76,7 @@
 
 	private boolean reverseOrder;
 	private List<CommitFilter> filters;
```

Comment:
Should rename the `filters` to `commitFilters` for consistency with `diffFilters`.